config.json

```diff
@@ -18,5 +18,9 @@
   },
   "proxy": "",
   "AWS_ACCESS_KEY_ID": "XXXXXXXXXXXXXXXXX",
-  "AWS_SECRET_ACCESS_KEY": "XXXXXXXXXXXXXXXXXXXXX"
+  "AWS_SECRET_ACCESS_KEY": "XXXXXXXXXXXXXXXXXXXXX",
```

Comment:
@aayusharora Yeah. There are many things(including these) which we should add on the README. I will add them in the next PR.